assets/js/src/bulk-delete.js

```diff
@@ -6,12 +6,46 @@
  * @author: Sudar <http://sudarmuthu.com>
  */
 
-/*global BulkWP, postboxes, pagenow*/
+/*global BulkWP, postboxes, pagenow, ajaxurl*/
 jQuery(document).ready(function () {
 	/**
 	 * Enable select2
 	 */
-	jQuery( '.select2' ).select2();
+	jQuery( '.select2' ).select2({
+		width: '300px'
```

Comment:
See if this can be included from css file.

Reply:
@sudar I could not set width using CSS, If i remove this line select2 automatically applied inline CSS